$
\newcommand{\p}[1]{\left(#1\right)}
\newcommand{\bk}[1]{\left[#1\right]}
\newcommand{\bc}[1]{ \left\{#1\right\} }
\newcommand{\abs}[1]{ \left|#1\right| }
\newcommand{\E}{ \text{E} }
\newcommand{\N}{ \mathcal N }
\newcommand{\ds}{ \displaystyle }
$

# 1a)

In [ ]:
set.seed(256)
jeans <- read.table("../dat/blue-jeans.txt",header=TRUE)
options(contrasts=c("contr.sum","contr.poly"))
mod <- lm(Scores ~ as.factor(Brand), data=jeans)
summary.mod <- summary(mod)
anova.mod <- anova(mod)

In [14]:
#1a)
summary.mod


Call:
lm(formula = Scores ~ as.factor(Brand), data = jeans)

Residuals:
     Min       1Q   Median       3Q      Max 
-0.89375 -0.40687  0.04125  0.35219  0.98000 

Coefficients:
                  Estimate Std. Error t value Pr(>|t|)    
(Intercept)        2.84344    0.09272  30.666   <2e-16 ***
as.factor(Brand)1 -0.41344    0.16060  -2.574   0.0156 *  
as.factor(Brand)2  0.39906    0.16060   2.485   0.0192 *  
as.factor(Brand)3  0.33031    0.16060   2.057   0.0491 *  
---
Signif. codes:  0 ‘***’ 0.001 ‘**’ 0.01 ‘*’ 0.05 ‘.’ 0.1 ‘ ’ 1

Residual standard error: 0.5245 on 28 degrees of freedom
Multiple R-squared:  0.3589,	Adjusted R-squared:  0.2902 
F-statistic: 5.225 on 3 and 28 DF,  p-value: 0.005434


In [15]:
anova.mod # this tells us if the factor Brand contributes to the model

Analysis of Variance Table

Response: Scores
                 Df Sum Sq Mean Sq F value   Pr(>F)   
as.factor(Brand)  3 4.3128 1.43761  5.2253 0.005434 **
Residuals        28 7.7035 0.27512                    
---
Signif. codes:  0 ‘***’ 0.001 ‘**’ 0.01 ‘*’ 0.05 ‘.’ 0.1 ‘ ’ 1

We obtained an F-statistic in the ANOVA table above. F = 5.225 $\Rightarrow$ p-val = .00543 < .05.

$\therefore$ We reject the null hypothesis that the jeans have no effect. Note that the output is testing

$$
\begin{cases}
    H_0: \text{Brands don't contribute to the model}\\
    H_a: \text{Brands contribute to the model}
\end{cases}
$$

The anova function is smart enough to recognize the Brand factor as **one** variable. In our classes notation,
the first row of the anoav output corresponds to $SS(\boldsymbol\alpha | \mu)$

# 1b)

Now, we want to test the hypothesis that brands 2 & 3 are cheap and brands 1 & 4 are expensive. In other words,
we're testing 

$$
    H_0: C\boldsymbol\alpha = 0,
$$

where $C = \p{
    \begin{array}{rrr}
    0 & 1 & -1 &  0 \\
    1 & 0 &  0 & -1 \\
    1/2 & -1/2 & -1/2 & 1/2 \\
    \end{array}
}$.

The $C$ matrix follows from the hypothesis that brands 2,3 are the same; 1,4 are the same; and 2,3 are different from 1,4.